css/bio.css

```diff
@@ -0,0 +1,73 @@
+.bio {
+  font-family: "Open Sans Condensed";
+  background-attachment: fixed;
+  margin-bottom: 18px;
+}
+
+.bio-pic {
+  width: 140px;
+  height: 140px;
+}
+
+.ellipsis-container {
+  position: relative;
+  padding-right: .5em;
+}
+
+.ellipsis {
+  background: white;
+  display: inline;
+  font-weight: normal;
+}
+
+.ellipsis:before,
+.ellipsis:after {
+  background: inherit;
+  position: absolute;
+}
+
+.ellipsis:before {
+  content: '...more >>';
+  font-weight: 600;
+  bottom: 0;
+  opacity: 1;
+  right: 0;
+  transition: .6s ease-in-out .3s;
+  visibility: visible;
+}
+
+.clamp-toggle:not( :checked ) + .ellipsis-container:hover > .ellipsis:before {
+  background: #fefefe;
+  color: #444;
+  padding: 0 .5em;
+}
+
+:nth-child(n):nth-child(n) .clamp-toggle:checked + .ellipsis-container > .ellipsis:before {
+  background: #fefefe;
+  opacity: 0;
+  visibility: hidden;
+  z-index: 1;
+}
+
+.ellipsis:after {
+  content: '';
+  height: 100%;
+  width: 100%;
+}
```

Comment:
I noticed a small bug on the volunteers page when the content is 12 columns wide but the bio is only two lines long.  The ellipsis:after element is width 100%, height: 100%, position: absolute, and it lays on top of the badges.  Here is a screen shot.
![screen shot 2016-07-06 at 8 46 35 pm](https://cloud.githubusercontent.com/assets/11672661/16640213/b58cb7c4-43bb-11e6-8554-953c136771ea.png)

Not sure of the best solution here.  And maybe when we get real bios (longer than two lines), we won't have to worry about this scenario.  So you decide how you prefer to handle.

I did discover that if `position: relative;` added to ellipsis:after on Line 56 instead, it seems to lay the More.../Less... on top of the badge, so that the user can see what's covering the badge and can dismiss it by clicking once.